graphql-compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/PluginConstants.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.graphql.compiler;
+
+/**
+ * GraphQl compiler plugin constants.
+ */
+public class PluginConstants {
+
+    // compiler plugin constants
+    public static final String PACKAGE_PREFIX = "graphql";
+    public static final String PACKAGE_ORG = "ballerina";
+
+    // resource function constants
+    public static final String RESOURCE_FUNCTION_GET = "get";
+
+    /**
+     * Compilation errors.
+     */
+    enum CompilationErrors {
```

Comment:
Can we add the `DiagnosticSeverity` to this enum since it is tightly coupled with the errors ?

Reply:
FIxed in 856ba10410bc7565b1e5e7adee3cb8273d33e4b3